ql/src/java/org/apache/hadoop/hive/ql/metadata/HiveStorageHandler.java

```diff
@@ -855,4 +855,8 @@ default MergeTaskProperties getMergeTaskProperties(Properties properties) {
     throw new UnsupportedOperationException("Storage handler does not support getting merge input files " +
             "for a table.");
   }
+
+  default void setCustomDeleteProperties(TableDesc tableDesc) {
```

Comment:
not sure about that table, maybe `setMergeTaskDeleteProperties`?

Reply:
Done.